src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/CommonInterests.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.NetCore.Analyzers.Runtime
+{
+    internal static class CommonInterests
+    {
+        internal static class JoinableTask
+        {
+            internal const string TypeName = "JoinableTask";
+            internal const string Join = "Join";
+            internal const string JoinAsync = "JoinAsync";
+        }
```

Comment:
I don't think all of this JoinableTask stuff should be here.  That's a specific library with its own set of analyzers.  This analyzer shouldn't need to know anything about JTF.